time TimeStamp
int32 Speed
int32 Angle
int32 Illumination
int32 AlleyLights
int32 TurningLights
bool Brake
bool Reset
bool EmergencyStop
bool EnabledEmergencyStop
int32 TimeThreshold_ms
int32 LeftWheel
int32 RightWheel
int32 DrivingMode
float32 AngularRate
float32 OrientationOffset
float32 OrientationSetpoint
int32 MovCamera
int32 cameraExp
int32 videoMode
uint64 UDPTimeStamp
uint64 QRTimeStamp
uint32 LastControlMsg
bool AutoPilot
bool Inferring
